<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txt_editora">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>174</y>
     <width>500</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>208</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ano:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cadsatrar">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>299</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Cadastrar </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>62</y>
     <width>500</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Noto Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Cadastrar Livro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>152</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Editora:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_voltar">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>299</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>96</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Autor:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_autor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>118</y>
     <width>500</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_ano">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>500</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome do livro:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
